src/common/ipc/ipc.ts

```diff
@@ -0,0 +1,146 @@
+// Inter-process communications (main <-> renderer)
+// https://www.electronjs.org/docs/api/ipc-main
+// https://www.electronjs.org/docs/api/ipc-renderer
+
+import { ipcMain, ipcRenderer, webContents, remote } from "electron";
+import { toJS } from "mobx";
+import { EventEmitter } from "ws";
+import logger from "../../main/logger";
+import { ClusterFrameInfo, clusterFrameMap } from "../cluster-frames";
+
+const subFramesChannel = "ipc:get-sub-frames";
```

Comment:
Shouldn't we prefix `subFramesChannel` with corresponding frame-id?

Reply:
Did I copy this over wrong (let me check)?

Reply:
This is copied directly from `master` and I haven't touched the code there.

Reply:
Why did we move `ipc` here?

Reply:
Do you mean why was `src/common/ipc.ts` was moved to `src/common/ipc/ipc.ts`? That is because I added another module file. This doesn't change any imports because of how `index.ts` is handled.

Reply:
Ok, asked because it's quite hard to spot differences when whole file is moved and modified 🙂

Reply:
Fair, should I move the argument checking functions to a different file so that this file is just a rename in the git diff?

Reply:
Nvm, I'll do that anyway